pipeline-scripts/release.groovy

```diff
@@ -167,9 +181,9 @@ def stageSetClientLatest(from_release_tag, arch, client_type) {
 
 }
 
-def stageTagRelease(quay_url, release_tag, arch) {
+def stageTagRelease(quay_url, release_name, release_tag, arch) {
     def archSuffix = getArchSuffix(arch)
-    def cmd = "GOTRACEBACK=all ${oc_cmd} tag ${quay_url}:${release_tag} ocp${archSuffix}/release${archSuffix}:${release_tag}"
+    def cmd = "GOTRACEBACK=all ${oc_cmd} tag ${quay_url}:${release_name} ocp${archSuffix}/release${archSuffix}:${release_tag}"
```

Comment:
I think the quay pullspec here should also use release_tag instead of release name?

Reply:
Yes, I think this actually got it backward; we are trying to tag the quay image (should be `release_tag`) into the imagestream (which being arch-specific already should just require the `release_name` as the tag).